<?xml version="1.0"?>
<launch>

  <rosparam file="$(find lidar_obstacle_detector)/cfg/rslidar.yaml" command="load" />
  <node name="obstacle_detector_node" pkg="lidar_obstacle_detector" type="obstacle_detector_node" output="screen">
    <!-- Input Topic Names -->
    <param name="lidar_points_topic"                  value="/lidar_odom_node/meta_cloud"/>  
    <!-- Output Topic Names -->
    <param name="cloud_ground_topic"                  value="obstacle_detector/cloud_ground"/>
    <param name="cloud_clusters_topic"                value="obstacle_detector/cloud_clusters"/>
    <param name="jsk_bboxes_topic"                    value="obstacle_detector/jsk_bboxes"/>
    <param name="autoware_objects_topic"              value="/autoware_tracker/cluster/objects"/>
    <!-- Parameters -->
    <param name="bbox_target_frame"                   value="base_link"/> 
  </node>

  <!-- Dynamic Reconfigure GUI -->
  <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" output="screen" />
  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find lidar_obstacle_detector)/rviz/lidar_obstacle_detector.rviz" output="log" respawn="true" />


</launch>
